src/Test.UnitTests.Sarif/Readers/DeferredCollectionsTests.cs

```diff
@@ -126,6 +156,34 @@ private static void CompareReadNormalToReadDeferred(string filePath)
             Assert.Equal(contextsCopy.Count, valueCount);
         }
 
+        private static void CompareReadNormalToReadDeferred(string filePath)
+        {
+            LogModelSampleBuilder.EnsureSamplesBuilt();
+            var serializer = new JsonSerializer();
+
+            Log expected;
+            Log actual;
+
+            // Read normally (JsonSerializer -> JsonTextReader -> StreamReader)
+            using (var reader = new JsonTextReader(new StreamReader(filePath)))
```

Comment:
why do you use the new using declaration syntax elsewhere but not here?

Reply:
changed and renamed the variables to be able to do that.